tests/framework/common.go

```diff
@@ -566,27 +566,34 @@ func (td *OsmTestData) InstallOSM(instOpts InstallOSMOpts) error {
 
 	var args []string
 	args = append(args, "install",
-		"--container-registry="+instOpts.ContainerRegistryLoc,
-		"--osm-image-tag="+instOpts.OsmImagetag,
 		"--osm-namespace="+instOpts.ControlPlaneNS,
-		"--certificate-manager="+instOpts.CertManager,
-		"--enable-egress="+strconv.FormatBool(instOpts.EgressEnabled),
-		"--enable-permissive-traffic-policy="+strconv.FormatBool(instOpts.EnablePermissiveMode),
-		"--enable-debug-server="+strconv.FormatBool(instOpts.EnableDebugServer),
-		"--envoy-log-level="+instOpts.EnvoyLogLevel,
+		fmt.Sprintf("--timeout=%v", 90*time.Second),
+	)
+
+	instOpts.SetOverrides = append(instOpts.SetOverrides,
+		fmt.Sprintf("OpenServiceMesh.image.registry=%s", instOpts.ContainerRegistryLoc),
+		fmt.Sprintf("OpenServiceMesh.image.tag=%s", instOpts.OsmImagetag),
+		fmt.Sprintf("OpenServiceMesh.certificateManager=%s", instOpts.CertManager),
+		fmt.Sprintf("OpenServiceMesh.enableEgress=%s", strconv.FormatBool(instOpts.EgressEnabled)),
```

Comment:
Nit: I would either use concatenation with `+` and not `fmt.Sprintf` or use `%v/%t` in the format string so then the `strconv.FormatBool` isn't needed.

So either

```go
"OpenServiceMesh.someBool=" + strconv.FormatBool(someBool),
```

or

```go
fmt.Sprintf("OpenServiceMesh.someBool=%v", someBool),
```

Reply:
Ahh, good catch - it's fixed now.